appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -746,6 +758,19 @@ public PayaraMicroImpl setNoCluster(boolean noCluster) {
         return this;
     }
 
+    /**
+     * Enables or disables clustering before bootstrap
```

Comment:
Not just clustering, Hazelcast in general.
```suggestion
     * Enables or disables Hazelcast
```

Reply:
As per FISH-372, the option --nohazelcast does what the option --nocluster used to do in the previous release, i.e. enable or disbale the clustering.